sdk/pinocchio/src/lib.rs

```diff
@@ -228,6 +227,10 @@ pub use solana_account_view::AccountView;
 pub use solana_address as address;
 pub use solana_address::Address;
 
+// Re-export the `solana_define_syscall` for downstream use.
+#[cfg(target_os = "solana")]
```

Comment:
To go with the other change:
```suggestion
#[cfg(any(target_os = "solana", target_arch = "bpf"))]
```